lib/config/presets/internal/schedule.ts

```diff
@@ -21,6 +21,10 @@ export const presets: Record<string, Preset> = {
     description: 'Schedule quarterly',
     schedule: ['every 3 months on the first day of the month'],
   },
+  yearly: {
+    description: 'Schedule once a year',
```

Comment:
```suggestion
    description: 'Schedule once a year (think before you use this preset, you'll miss critical security updates)',
```

Reply:
I think so, but maybe just use `(not recommended)` ?